site-packages/integralstor_utils/ntp.py

```diff
@@ -29,4 +30,33 @@ def get_ntp_servers():
         return list, None
 
 
+def sync_ntp():
+    return_dict = {}
+    try:
+        ntp_servers, err = get_ntp_servers()
+        if err:
+            raise Exception(err)
+        output, err = services_management.update_service_status('ntpd', 'stop')
+        if err:
+            raise Exception(err)
+        for server in ntp_servers:
+            cmd = "ntpdate -b %s" % server
+            output, err = command.get_command_output(cmd, shell=True)
+            if err:
+                continue
+            else:
+                return_dict['ntp_sync'] = True
+                return_dict['server_used'] = server
+                break
+        output, err = services_management.update_service_status(
+            'ntpd', 'start')
+        if err:
+            raise Exception(err)
+
+    except Exception, e:
+        return None, 'Could not sync time with ntp servers please configure some ntp servers'
```

Comment:
Pl change the exception message to : Could not sync time with ntp servers. Please check or reconfigure the ntp servers